spec/helpers/application_formatters_helper_spec.rb

```diff
@@ -24,6 +24,51 @@
         expect(helper.format_html('<script/>')).to eq('')
       end
 
+      it 'does not remove span tags' do
+        html = <<-HTML
+          <span>Hello World!</span>
```

Comment:
If the strings are short, can just do `html = '<span>Hello World!</span>'` instead of using here doc.